rfcs/webtransport_h3_test_server.md

```diff
@@ -0,0 +1,88 @@
+# RFC 85: WebTransport over HTTP/3 Test Server
+
+## Summary
+
+Add a [WebTransport over HTTP/3](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3-01) server to wpt. The server is used to test the [WebTransport Web APIs](https://w3c.github.io/webtransport/).
+
+The server will use a new [file name flag](https://web-platform-tests.org/writing-tests/file-names.html) `.webtransport` (`.webtransport.py`) for handlers. Handlers are python scripts which are used to respond WebTransport events.
+
+This RFC shares the same background/motivation with [RFC 42](https://github.com/web-platform-tests/rfcs/blob/master/rfcs/quic.md) but focuses on WebTransport over HTTP/3. See RFC 42 for more background.
+
+## Details
+
+### Implementation
+
+The WebTransport over HTTP/3 test server is built on top of `aioquic`, which is already added under `tools/third_party`. As of version 0.9.11, `aioquic` lacks some mechanisms to implement a WebTransport server (e.g. configuring SETTINGS HTTP/3 frame) but we plan to update `aioquic` once it supports necessary mechanisms.
+
+The server will be implemented under `tools/webtransport` directory.
+
+### Handlers
+
+A WebTransport handler is a Python script which contains a function `handle_event()`. Here is an example handler which echos back received data.
+
+```python
+from webtransport import (
+    WebTransportSession,
+    BidirectionalStreamDataReceived,
+    DatagramReceived,
+    WebTransportEvent
+)
+
+def handle_event(session: WebTransportSession,
```

Comment:
Is session establishment an event? [tools/quic](https://github.com/web-platform-tests/wpt/tree/master/tools/quic) had a different function for the client indiciation.

Reply:
Yes, in this approach. Updated the [document](https://bashi.github.io/wpt-wt-test-server-api/events.html#webtransport.events.WebTransportConnectReceived).

In the alternative approach, we would have a function for session establishment.